app/models/miq_worker_type.rb

```diff
@@ -0,0 +1,22 @@
+class MiqWorkerType < ApplicationRecord
+  EXCLUDED_CLASSES = [ManageIQ::Providers::BaseManager::OperationsWorker].freeze
```

Comment:
Since this is defined at the class level, this forces loading the operations worker early even if it's not necessary.  Might be better to make it a string.  Then again, it's usage is happening at seed, which is early anyway, so I'm not sure

(Gut reaction-response to a constant at the class level :smile: )

Reply:
Yeah, so I could either make this a string then constantize all the elements in the method or just define the array in the method each time it's called ... thoughts?

Reply:
I was thinking String then constantize...it's up to you.